### ISSUE TITLE
Univariate timeseries regression

### ISSUE THREAD

**Issue by gaImdea** [shap/shap#2402]
Hi ! I am quite new to SHAP.
I would like to know if  DeepExplainer can be used to explain timeseries  wih one feature only, e.g. temperature. Let's suppose I have an LSTM with an history of 10 timesteps and I want to predict the temperature value at the 11th timestep. I would like to get the shap values for each timestep, i.e, t-10,t-9,t-8,t-1 for each sample. I am using the same approach for Sentiment Analysis with Logistic Regression https://shap.readthedocs.io/en/latest/example_notebooks/tabular_examples/linear_models/Sentiment%20Analysis%20with%20Logistic%20Regression.html

so basically what I am doing is :

explainer = shap.DeepExplainer(model, train_x)
--explain the first 10 predictions
shap_values = explainer.shap_values(test_x_[:10])
--plot first prediction explanations
shap.force_plot(explainer.expected_value[0], shap_values[0][0], test_x[0])

Does this approach makes sense?
Thanks a lot!
